plugin/src/main/java/com/denizenscript/denizen/objects/BiomeTag.java

```diff
@@ -201,28 +209,37 @@ public static void registerTags() {
         // @mechanism BiomeTag.humidity
         // @description
         // Returns the humidity of this biome.
+        // @Example
+        // - narrate "Humidity is currently <player.location.biome.humidity>! So humid!"

```

Comment:
Should specify an example of the expected result, eg `# If you're standing in a Plains biome this fills with '0.5'` or something

Reply:
I added this to a few other related tags. Like `<BiomeTag.downfall_type>`, `<BiomeTag.humidity>`, `<BiomeTag.temperature>`